### PR TITLE
Stabilize behat by overriding method of MinkContext with spin

### PR DESCRIPTION
**Pull request by ahocquard**
**Description (for Contributor and Core Developer)**
Use spin + delete dead code.

Should fix random on tests/legacy/features/user/edit_user_group_and_role_assignations.feature:33 on master

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: behat.yml.dist [hunk 1]
```diff
@@ -6,9 +6,7 @@ default:
             filters:
                 tags: "~@skip&&~@skip-pef&&~@skip-nav&&~@doc&&~@unstable&&~@unstable-app&&~@deprecated"
             contexts:
-                - Behat\MinkExtension\Context\MinkContext
                 - Context\FeatureContext:
-                    - 'Context\FeatureContext'
```
Comment (jjanvier) [inline behat.yml.dist]: what was this line :D

File: features/Context/FeatureContext.php [hunk 2]
```diff
@@ -37,7 +36,7 @@
  * @copyright 2013 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-class FeatureContext extends PimContext implements KernelAwareContext
+class FeatureContext extends MinkContext implements KernelAwareContext
```
Comment (jjanvier) [inline features/Context/FeatureContext.php]: is PimContext still used?

Comment (ahocquard) [reply features/Context/FeatureContext.php]: Yes, by all a lot of context. A nightmare to to know which context uses what...

File: behat.yml.dist (hunk 1 above)
Comment (ahocquard) [reply behat.yml.dist]: it allows to get the main context, which is basically FeatureContext, so itself 🎉

**Pull request merged**